resources/config/snemo/demonstrator/simulation/vertexes/4.1/generators/source_vg.def

```diff
@@ -164,8 +164,12 @@ generators : string[2] = "source_pads_internal_surface" "source_pads_external_su
 # relative masses of internal/external strips:
 #
 #@variant_if    geometry:layout/if_basic/source_layout/if_basic|true
-generators.source_pads_external_bulk.absolute_weight : real  =  250.0 # 125 mm * 2
-generators.source_pads_internal_bulk.absolute_weight : real  = 4607.0 # 135.5 mm * 34
+# Date:    2017-10-20
+# Author:  frc
+# Issue:   https://github.com/SuperNEMO-DBD/Falaise/issues/59
+# Comment: Copy/paste error on combined generators' names.
+generators.source_pads_external_surface.absolute_weight : real  =  250.0 # 125 mm * 2
+generators.source_pads_internal_surface.absolute_weight : real  = 4607.0 # 135.5 mm * 34
 #@variant_endif geometry:layout/if_basic/source_layout/if_basic
```

Comment:
Please add a test of running these options in `flsimulate` - see the files and CMake script in source/flsimulate/tests for examples.